src/fn/stampboard.ts

```diff
@@ -165,7 +165,7 @@ export const stampboard = (
         holes.push(
           platedhole(
             i + 1,
-            params.w / 2 - params.pl + outerDiameter / 2,
+            params.w / 2 - 1.61,
```

Comment:
Is this intentionally hard coded? Shouldnt it be derived or a default?

Reply:
<img width="645" alt="Screenshot 2024-10-21 at 11 47 49 PM" src="https://github.com/user-attachments/assets/ae19645c-d135-4832-b7be-88ce9d5f94ab">
I write it hard coded to be like the data sheet
if I need it to be dynamic then I need to modify `params.pl + outerDiameter / 2`

Reply:
oh we just need to give a name to that distance and set the default to 1.61. That parameter should be called `innerholeedgedistance` i think

Reply:
<img width="517" alt="Screenshot 2024-10-22 at 12 00 15 AM" src="https://github.com/user-attachments/assets/9175139d-17fe-441c-882e-1f4b640af70f">

Reply:
yep!